docs/remote-config/_get-started.md

```diff
@@ -18,6 +18,10 @@ sample code.
     you <a href="https://support.google.com/firebase/answer/9289399#linkga"
            class="external">enable Google Analytics</a> in your project.
 
+1.  When you use Remote Config for macOS, you need to the add the [Keychain
+    Sharing](https://developer.apple.com/documentation/xcode/configuring-keychain-sharing)
+    capability in XCode.
```

Comment:
@kevinthecheung - could you let me know if this docs change is acceptable? Thanks.

Reply:
@russellwheatley I have added it as a separated. But now there are two note boxes below each other. Not sure if there are style guide which address this problem differently.

```
Note: Because the Remote Config SDK has a dependency on the Remote Config REST
API, make sure that you do **not** disable that API, which is enabled by default
in a typical project.

Note: When you use Remote Config for macOS, you need to the add the [Keychain
Sharing](https://developer.apple.com/documentation/xcode/configuring-keychain-sharing)
capability in XCode.
```